pac/k8s/net/azsdkdemonetapi-deployment.yaml

```diff
@@ -15,123 +14,129 @@ spec:
+              key: AZURE_COSMOS_DB
+              name: env
+        - name: AZURE_COSMOS_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_COSMOS_ENDPOINT
+              name: env
+        - name: AZURE_COSMOS_KEY_NAME
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_COSMOS_KEY_NAME
+              name: env
+        - name: AZURE_FORM_RECOGNIZER_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_FORM_RECOGNIZER_ENDPOINT
+              name: env
+        - name: AZURE_KEYVAULT_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_KEYVAULT_ENDPOINT
+              name: env
+        - name: AZURE_STORAGE_BLOB_CONTAINER_NAME
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_BLOB_CONTAINER_NAME
+              name: env
+        - name: AZURE_STORAGE_BLOB_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_BLOB_ENDPOINT
+              name: env
+        - name: AZURE_STORAGE_QUEUE_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_QUEUE_ENDPOINT
+              name: env
+        - name: AZURE_STORAGE_QUEUE_MSG_COUNT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_QUEUE_MSG_COUNT
+              name: env
+        - name: AZURE_STORAGE_QUEUE_NAME
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_QUEUE_NAME
+              name: env
+        - name: AZURE_STORAGE_QUEUE_RECEIVE_SLEEP
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_STORAGE_QUEUE_RECEIVE_SLEEP
+              name: env
+        - name: AZURE_TENANT_ID
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_TENANT_ID
+              name: env
+        - name: AZURE_TEXT_ANALYTICS_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: AZURE_TEXT_ANALYTICS_ENDPOINT
+              name: env
+        - name: AZURITE_ACCOUNT_KEY
+          valueFrom:
+            configMapKeyRef:
+              key: AZURITE_ACCOUNT_KEY
+              name: env
+        - name: AZURITE_ACCOUNT_NAME
+          valueFrom:
+            configMapKeyRef:
+              key: AZURITE_ACCOUNT_NAME
+              name: env
+        - name: MEME_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: MEME_ENDPOINT
+              name: env
+        image: vyta/azsdkdemonetapi
+        imagePullPolicy: "IfNotPresent"
+        name: azsdkdemonetapi
+        ports:
+        - containerPort: 80
+        - containerPort: 443
+        resources: {}
+        volumeMounts:
+        - mountPath: /root/.azure
+          name: azsdkdemonetapi-hostpath0
       restartPolicy: Always
       serviceAccountName: ""
-      volumes: null
+      volumes:
+      - hostPath:
+          path: /home/vyta/.azure
```

Comment:
I'm assuming .yaml is not configurable...wondering how to make this work without vyta